package.json

```diff
@@ -29,7 +29,7 @@
     "@vue/test-utils": "^1.0.0-beta.29",
     "chai": "^4.2.0",
     "chai-spies": "^1.0.0",
-    "node-sass": "^4.12.0",
+    "node-sass": "^4.14.1",
```

Comment:
Was this a blocker for you or was it an unintended change here? I think this is the change that is causing the conflict.

Reply:
I ran `npm audit fix` out of habit, I don't think there's any breaking changes but I'll revert it for now.